skymp5-functions-lib/src/logic/TestUtils.ts

```diff
@@ -29,8 +30,18 @@ export const makePlayerController = (): PlayerController => {
     getActorDistanceSquared(actorId1, actorId2) {
         return 1
     },
-    getSetting(name) {
-        return 2000
+    getServerSetting(name: string) {
+      const settings: ChatSettings = {
+        hearingRadiusNormal: 2000,
+        whisperDistance: 0.1,
+        shoutDistance: 1.5,
+        minDistanceToChange: 500,
+      }
+      if (name in settings) {
+        return settings[(name as keyof ChatSettings)]
+      } else {
+        return 1
```

Comment:
should be some exception I think
`;` also

Reply:
I've rewrote the typing of the settings a bit